src/EvoSC.Manialinks/Templates/Scripts/Window.ms

```diff
@@ -0,0 +1,109 @@
+﻿<!--
+Void CloseWindow(Text windowId) {
+    declare Window = Page.MainFrame.GetFirstChild(windowId) as CMlFrame;
+    Window.Hide();
+    
+    +++ OnWindowClosed +++
```

Comment:
We should introduce a global script variable to trigger exiting the script. Even if the main-frame is hidden, the script will continue to run doing yields forever, unless we call a `return;` in the main-loop.